include/core/mortar_coupling_manager.h

```diff
@@ -319,12 +335,13 @@ class MortarManager
 
 private:
   /**
-   * @brief Verifies whether the mesh is aligned
+   * @brief Returns the mesh alignement type and cell index
+   * @param[in] rad Angular coordinate of cell center
    * @param[out] type Cell configuration type at the interface
    *                  type = 0: mesh aligned
-   *                  type = 1: mesh not aligned, points are closer than a
-   * tolerance type = 2: mesh not aligned, points are in rotated configuration
-   * @param[out] id Point index
+   *                  type = 1: mesh not aligned, inner domain (allows rotation)
+   *                  type = 2: mesh not aligned, outer domain (fixed)
+   * @param[out] id Index of the cell in which lies the rotated cell center
```

Comment:
I would modify this part also to have return :). Especially since you are returning a pair :)